tests/framework/installer/install_helper.go

```diff
@@ -156,13 +159,11 @@ func (h *InstallHelper) CreateK8sRookCluster(namespace string, storeType string)
 func (h *InstallHelper) CreateK8sRookClusterWithHostPathAndDevices(namespace, storeType, dataDirHostPath string,
 	useAllDevices bool, mon cephv1alpha1.MonSpec, startWithAllNodes bool) error {
 
-	ns := &v1.Namespace{ObjectMeta: metav1.ObjectMeta{Name: namespace}}
-	_, err := h.k8shelper.Clientset.CoreV1().Namespaces().Create(ns)
-	if err != nil && !errors.IsAlreadyExists(err) {
-		return fmt.Errorf("failed to create namespace %s. %+v", namespace, err)
+	if err := h.k8shelper.CreateNamespace(namespace); err != nil {
+		return err
 	}
 
-	if h.k8shelper.VersionAtLeast("v1.8.0") {
```

Comment:
The version here doesn't really matter, but was more about a way to predictably hit both the clientset and the yaml test paths. These two code paths didn't really provide much benefit though. We should probably just stick with the yaml and get rid of the clientset implementation so we can more easily mirror in the tests what is happening in the sample yaml.

Reply:
i'm okay with doing YAML only so that it more closely mirrors the sample path that a user would take.